runners/console/index.ts

```diff
@@ -143,6 +143,19 @@ export class Console extends Runner {
         return result;
     }
 
+    runCloneRepository(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let workspaceDir = path.join(this.getWorkingDirectory(), "devonfw", "workspaces", "main");
+        let directorypath = path.join(workspaceDir, command.parameters[0]);
+        
+        this.createFolder(directorypath, false);
```

Comment:
If you changed the second parameter to 'true', you will probably not get any error messages when building the project and the folder already exists.

Reply:
Do we have to change createFile and changeFile, too? They are also written to be solely used in workspaceDir.

Reply:
That's right, then we have to change this too. I think the command should work even without a Devon Ide. @maybeec  what do you think?